meshtastic/interfaces/ble/client.py

```
@@ -158,28 +182,225 @@ def __init__(
+            Operation name included in fallback error logging.
+
+        Returns
+        -------
+        None
+            Always returns ``None``.
+        """
+        safe_cleanup = self._resolve_error_handler_hook("safe_cleanup", "_safe_cleanup")
+        if safe_cleanup is not None:
+            try:
+                safe_cleanup(cleanup, cleanup_name=operation_name)
+                return
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)